src/servers/src/prometheus.rs

```diff
@@ -355,10 +356,13 @@ fn to_grpc_insert_request(database: &str, mut timeseries: TimeSeries) -> Result<
     }
 
     Ok(GrpcInsertRequest {
-        schema_name,
-        table_name: table_name.context(error::InvalidPromRemoteRequestSnafu {
-            msg: "missing '__name__' label in timeseries",
-        })?,
+        full_tablename: Some(FullTableName {
+            catalog_name: DEFAULT_CATALOG_NAME.to_string(),
```

Comment:
It's better to avoid using default catalog here. We can extract catalog from database name using `server::parse_catalog_and_schema_from_client_database_name`

Reply:
Same as above.